slam_node:
  ros__parameters:
    points_topic: "/tree_points"
    cylinder_topic: "/landmark_trunks"
    robot_vis_topic: "/robot_cov"
    landmark_vis_topic: "/landmarks"